projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -117,7 +113,7 @@ private function hide_customizer_menu_on_block_theme() {
 
 			remove_action( 'customize_register', array( 'Jetpack_Fonts_Typekit', 'maybe_override_for_advanced_mode' ), 20 );
 
-			remove_action( 'customize_register', 'Automattic\Jetpack\Dashboard_Customizations\register_css_nudge_control' );
+			remove_action( 'customize_register', 'Automattic\Jetpack\Masterbar\register_css_nudge_control' );
```

Comment:
This is currently not working for Simple sites since we are still using the module code on WPCOM. We should probably add a `class_exists` here so that when we are on WPCOM (Simple sites) we keep using `Automattic\Jetpack\Dashboard_Customizations\register_css_nudge_control`, while on remote sites (self-hosted and WoA) we use `Automattic\Jetpack\Masterbar\register_css_nudge_control`

Reply:
Done via https://github.com/Automattic/jetpack/pull/37676/commits/6aa5a1e00db928a3661550aeed4eb5a1f75b3076